webapp/channels/build/emoji/make_emojis.mjs

```diff
@@ -456,12 +453,15 @@ ${cssCats.join('\n')}
 ${cssEmojis.join('\n')}
 `;
 
-// write emoji.ts
-endResults.push(writeFile('_emojisprite.scss', 'sass/components/_emojisprite.scss', cssRules));
+// Create the emojisprite.scss file
+const emojispriteDir = path.resolve(webappRootDir, 'channels', 'src', 'sass', 'components', '_emojisprite.scss');
+endResults.push(writeToFileAndPrint('_emojisprite.scss', emojispriteDir, cssRules));
 
 Promise.all(endResults).then(() => {
-    console.log(warnLogColor, '\n[WARNING] Remember to run `make i18n-extract` as categories might have changed.');
+    console.log(warnLogColor, '\n[WARNING] Remember to npm run \'i18n-extract\' as categories might have changed.');
+    console.log(warnLogColor, '[WARNING] Remember to run `go generate ./server/public/model/emoji_data.go` from the root of the repository.');
```

Comment:
And could you just change this to tell the user to run the `gofmt -w` command?

Reply:
added the comment